README.md

```diff
@@ -1,45 +1,75 @@
-# Catapult-explorer
+# catapult-explorer
 
-## Setup Client
-```
-cd client
-npm install
-```
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-#### Compiles and hot-reloads for development
-```
-cd client
-npm run serve
-```
+The nem2 Command Line Interface (CLI) is a unified tool to interact with the NEM2 Blockchain.
 
-#### Compiles and minifies for production
-```
-cd client
-npm run build
-```
+This tool will enable you perform the most common and regular used actions in NEM2 (a.k.a Catapult)
+
+
+
+## Requirements
+
+The catapult-explorer works on NodeJS versions:
+
+- 8.9.X
+- 9.X.X
```

Comment:
Current [LTS](https://nodejs.org/) is 10.16.X, consider updating the project to make it compatible.